fontc_crater/src/args.rs

```diff
@@ -26,21 +26,22 @@ pub(super) struct RunArgs {
     /// Reusing this directory saves us having to clone all the repos on each run.
     ///
     /// This directory is also used to write cached results during repo discovery.
+    #[arg(default_value = "./font_repos")]
     pub(super) font_cache: PathBuf,
     /// Path to local checkout of google/fonts repository
-    #[arg(short, long)]
+    #[arg(short, long, default_value = "../fonts")]
```

Comment:
is this documented anywhere? feels like a potential footgun to decide we own some arbitrary sibling directory named 'fonts'

Reply:
My naive expectation was `--help` would reflect the default. It's a pretty natural place for it to land.

Reply:
i mean it's a reasonable guess, but it still feels sketchy?

bear in mind that,

- if you don't pass anything we'll just use a transient tempdir
- this arg only impacts the non-ci behaviour

Reply:
google/fonts is relatively heavy, I don't think re-pullking it all the time is a good default. I changed the default to under font_repos which is now gitignore'd.

Reply:
just want to point out that for CI you only fetch this repo when you're making the input list, which you do rarely.